pouta_blueprints/utils.py

```diff
@@ -121,3 +132,47 @@ def parse_port_range(port_range):
             raise ValueError('Port range invalid')
     else:
         raise ValueError('No port range found')
+
+
+def get_full_blueprint_config(blueprint):
+
+    template = blueprint.template
+    allowed_attrs = template.allowed_attrs
+    allowed_attrs = ['name', 'description'] + allowed_attrs
+    full_config = template.config
+    bp_config = blueprint.config
+    for attr in allowed_attrs:
+        if attr in bp_config:
+            full_config[attr] = bp_config[attr]
+    return full_config
+
+
+def get_blueprint_fields_from_config(blueprint, field_name):
+
+    full_config = get_full_blueprint_config(blueprint)
+
+    if field_name == 'preallocated_credits':
+        preallocated_credits = False  # Default value
```

Comment:
should we maybe store the defaults in a dict or a namespace so that they're all in one place?